docs/distributed_instructions.md

```diff
@@ -0,0 +1,43 @@
+# Remote Job Scheduling with Dask and Prefect
+
+We are working on a better, easier, and faster implementation to set all of this up. For now, follow the steps below:
+
+##  Run the Dask scheduler on the SLURM cluster
+0. You have installed this package on the SLURM cluster (and children nodes).
```

Comment:
I would be more specific than this personally.

0. Create an environment on the SLURM cluster / isilon and run:
```bash
pip install -e .[distributed]
```